modules/ipi-install-creating-an-rhcos-images-cache.adoc

```diff
@@ -15,6 +15,11 @@ The installation program no longer needs the `clusterOSImage` {op-system} image
 
 If you are running the installation program on a network with limited bandwidth and the {op-system} images download takes more than 15 to 20 minutes, the installation program will timeout. Caching images on a web server will help in such scenarios.
 
+[WARNING]
+====
+If you enable TLS for the HTTPD server, confirm the root certificate is signed by an authority trusted by the client and verify the trusted certificate chain between your {product-title} hub and spoke clusters and the HTTPD server. Using a server configured with an untrusted certificate will prevent the images from being downloaded to the image creation service. Using untrusted HTTPS servers is not supported.
```

Comment:
```suggestion
If you enable TLS for the HTTPD server, you must confirm the root certificate is signed by an authority trusted by the client and verify the trusted certificate chain between your {product-title} hub and spoke clusters and the HTTPD server. Using a server configured with an untrusted certificate prevents the images from being downloaded to the image creation service. Using untrusted HTTPS servers is not supported.
```
Just a couple of suggestions to enforce that the step is mandatory. Also, the ISG recommends avoiding the future tense when ever possible.